common/persistence/cassandra/cassandraPersistenceUtil.go

```diff
@@ -0,0 +1,1709 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
can you just name this file util.go ? cassandra/cassandraPersistenceUtil.go is redundant

Reply:
this is only for execution table, i.e. cassandraPersistence's helper function, not visibility

Reply:
btw, after this, i plan to also refactor the remaining ones, i.e. create workflow / continue as new and reset workflow execution, before 3DC reset mutable state change